js/src/analysis/classicEditorData.js

```diff
@@ -122,8 +132,27 @@ class ClassicEditorData {
 			return;
 		}
 
+		if ( ! forReplaceVariable ) {
+			switch( targetField ) {
```

Comment:
there should be a space after `switch` see also https://github.com/Yoast/eslint/pull/7